build/images/scripts/start_ovs_netdev

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+source logging
+source daemon_status
+
+CONTAINER_NAME="antrea-ovs"
+OVS_DB_FILE="/var/run/openvswitch/conf.db"
+
+set -euo pipefail
+
+hwaddr=$(ip link show eth0 | grep link/ether | awk '{print $2}')
+inet=$(ip addr show eth0 | grep "inet " | awk '{ print $2 }')
+gw=$(ip route | grep default | awk '{ print $3 }')
+
+# Modify ovs-ctl so that the kernel module is no longer loaded since it is not
+# needed when using OVS in userspace mode. It also enables running OVS with the
+# netdev datapath type on platforms which do not have the OVS kernel module.
+# This is easier than starting daemons manually...
+function fix_ovs_ctl {
+    sed -i 's/\(\w*\)\(insert_mod_if_required || return 1\)/\1# \2/' /usr/share/openvswitch/scripts/ovs-ctl
+}
+
+# See http://docs.openvswitch.org/en/latest/howto/userspace-tunneling/
+function add_br_phy {
```

Comment:
I did not read the link yet, but we do not need to connect br-int to this bridge in some way?

Reply:
This is actually IP forwarding. I believe it is taken care of either by OVS or Linux.